docs/authentication/configuration/restrictions.mdx

```diff
@@ -1,91 +1,79 @@
-
-For example, if you add "john.doe@clerk.dev" as a blocked email address, it means that anybody with "john.doe@clerk.dev" email address will not be able to sign up for your application but in addition "john.doe+anything@clerk.dev" will be blocked as well.
+For additional security, adding an individual email address to the blocklist will also block any attempts to sign up with the email address modified to contain a subaddress. Subaddresses are identified by the presense of any of the following characters in the local part of the email address: `+`, `#`, `=`.
 
-Subaddresses are identified by the presense of any of the following characters in the local part of the email address: `+`, `#`, `=`.
+For example, if you add `john.doe@clerk.dev` as a blocked email address, it means that anybody with `john.doe@clerk.dev` email address will not be able to sign up for your application, including `john.doe+anything@clerk.dev` and any other subaddress.
 
 ## Other restrictions
 
 ### Block email subaddresses
 
-The Block email subaddresses feature allows you to control who can get access to your application. It can restrict email addresses that contain the characters `+`, `=` or `#` from signing up or being added to existing accounts.
+The block email subaddresses feature allows you to block all email addresses that contain the characters `+`, `=` or `#` from signing up or being added to existing accounts. For example, an email address like `user+sub@clerk.com` will be blocked.
 
 <Callout type="info">
   Existing accounts with email subaddresses will not be affected by this restriction, and will still be allowed to sign in.
 </Callout>
 
-To enable this feature, in the Clerk Dashboard, go to **User & Authentication > [Restrictions](https://dashboard.clerk.com/last-active?path=user-authentication/restrictions)** and go to the **Restrictions** section. Turn on the **Block email subaddresses** toggle.
+To enable this feature:
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com).
+2. In the navigation sidebar, select **User & Authentication > [Restrictions](https://dashboard.clerk.com/last-active?path=user-authentication/restrictions)**.
+3. In the **Restrictions** section, toggle on **Block email subaddresses**.
 
-<Images
-width={3024}
-height={1654}
-src="/docs/images/authentication/restrictions/subaddresses.jpg"
-alt="The 'Restrictions' section in the 'Restrictions' tab under the 'User & Authentication' section of the Clerk Dashboard. A red arrow is pointing to the 'Block email subaddresses' toggle, which is toggled on."
-/>
+### Block sign-ups that use disposable email addresses
 
-After turning on the Block email subaddresses feature, any sign ups with an email address that contains a subaddress will be blocked.
+The block disposable email addresses feature allows you to block all email addresses that are *known to be disposable* from signing up for your application. This is useful to prevent spam accounts from signing up.
```

Comment:
Is there any more information on what we consider "known to be disposable"? If there's a list of domains that we use, it'd be worth linking to it here. If not, maybe we can just elaborate on our logic for what's "disposable"

Reply:
Yes that's what I wanted to do, but not only do we have a place that we could link to but the list has over 165k entries - see slack discussion here https://clerkinc.slack.com/archives/C01L98VHLM6/p1690548173428209